File: Algoritmos Traduzidos/Perl/Algoritmos_Filipe_Dias/109.pl
print "Insira o peso dos peixes:\n";
$peso = <STDIN>;
if ($peso > 50)
{
	$ex = $peso - 50;
	$ex = $ex * 4;
	print "Tem de pagar $ex euros pelo excedente de peixe apanhado\n";
}
else
{
	print "Nao precisa de pagar multa\n";
}

File: Algoritmos Traduzidos/Ruby/Algoritmos_Filipe_Dias/109.rb
puts "Insira o peso dos peixes:"
peso = gets
if peso.to_i > 50
  ex = peso.to_i - 50
  ex = ex * 4
  puts "Tem de pagar #{ex} euros pelo excedente de peixe apanhado"
else
  puts "Nao precisa de pagar multa"
end